package.json

```diff
@@ -31,21 +31,21 @@
     "url": "https://github.com/snyk/dep-graph/issues"
   },
   "devDependencies": {
-    "@types/graphlib": "^2.1.6",
-    "@types/jest": "^25",
+    "@types/graphlib": "^2.1.7",
+    "@types/jest": "^26.0.23",
     "@types/node": "^8",
-    "@types/object-hash": "^1.3.1",
-    "@types/semver": "^7",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
+    "@types/object-hash": "^2.1.0",
+    "@types/semver": "^7.3.6",
+    "@typescript-eslint/eslint-plugin": "^4.28.0",
+    "@typescript-eslint/parser": "^4.28.0",
     "benny": "^3.6.15",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.11.0",
-    "jest": "^26",
-    "prettier": "^2.0.5",
-    "ts-jest": "^26",
-    "ts-node": "^8",
-    "typescript": "~3.8.3"
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "^8.3.0",
+    "jest": "^27.0.5",
+    "prettier": "^2.3.1",
+    "ts-jest": "^27.0.3",
+    "ts-node": "^10.0.0",
+    "typescript": "~4.3.4"
```

Comment:
Hmm.. I thought we were trapped on 3.x due to.. something weird that I can't remember

Reply:
Also, is it wise to upgrade `typescript` without upgrading `tslib`?

Reply:
I did consider this, but I'm not sure they're inherently coupled? I'm not changing the compile target

Reply:
> something weird that I can't remember

But maybe not an issue for dep-graph. I think it's something to do with imports/exports and mocking